bootstrap/syncmanager.go

```diff
@@ -26,6 +29,7 @@ const DEFAULT_MAX_RETRY_TIME = MAX_RETRY_TIME_ALWAYS_RETRY
 type SyncManager struct {
 	rdbConn              rdb.Conn
 	client               *tendermint.HTTPClient
```

Comment:
please help to update the naming to distinguish different clients

Reply:
changed from `client` => `tendermintClient`.